frontend/src/container/Header.jsx

```diff
@@ -117,13 +117,13 @@ class Header extends React.Component {
 
               {!unLoggedAllowedPageList.includes(props.location.pathname) && props.appList.some(a => a.slug === 'agenda') && (
                 <li className='header__menu__rightside__agenda'>
-                  <button
+                  <a
```

Comment:
Please use the <Link> component from react-router-dom.
see https://reacttraining.com/react-router/web/api/Link

It'll render a <a> tag and you will be able to remove the `handleClickAgendaButton` handler